fern/docs.yml

```diff
@@ -6,64 +6,167 @@ instances:
         owner: fern-api
         repo: fern
         branch: main
-  # - url: fern-beta.docs.buildwithfern.com/learn
-  #   custom-domain: staging.buildwithfern.com/learn
 title: Fern | Documentation
 
-tabs:
-  overview:
-    display-name: Overview
-    icon: "globe"
-  sdks:
-    display-name: SDKs
-    slug: sdks
-    icon: code
-  docs:
-    display-name: Docs
-    slug: docs
-    icon: book
-
-navigation:
-  - tab: overview
-    layout:
+navigation: 
+  - section: Home
+    contents: 
       - page: Introduction
-        path: ./docs/pages/overview/introduction.mdx
-      - page: Why Fern?
-        path: ./docs/pages/overview/why-fern.mdx
-      - page: Get started
-        path: ./docs/pages/overview/get-started.mdx
-      - section: Define your API
+        path: ./overview/intro.mdx
+  - section: SDKs
+    contents: 
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Features
```

Comment:
Splitting up the 15 features listed into sections will make them more digestible.


**Section**: Core SDK Features
- Strongly typed
- Idiomatic method names
- Schema Validation
- Auto-Pagination
- Retries with backoff
- Webhook Signature Verification
- Forward Compatibility
- Multipart Form Data
- Idempotency Headers
- Server-sent Events

**Section**: Automation and Integrations
- Augment with Custom Code [doesn't really fit here]
- Integration Tests
- Code Examples
- Merging multiple APIs
- GitHub integration
- Package manager integrations

----

What about semantic versioning of the SDKs?

Reply:
LGTM